mon-pix/app/styles/globals/_palette.scss

```diff
@@ -69,5 +69,6 @@ $butterfly-bush-dark: #5E2563;
   Gradients
  */
 $pix-blue-to-purple-gradient: linear-gradient(to right, $dogder-blue, $purple);
+$pix-emerald-to-emerald-gradient: linear-gradient(to right, $emerald, $emerald-dark);
```

Comment:
je trouve pas très clair le nom de ce gradient. À la lecture on croit que c'est la même couleur. pourquoi pas `pix-emerald-to-emerald-dark-gradient` ?